Gemfile

```diff
@@ -41,6 +41,10 @@ gem 'rake', '10.4.1'
 
 gem 'sass-rails', '4.0.3'
 
+group :production do
```

Comment:
This should be in `assets`: the command failing is:

    rake#RAILS_ENV=production RAILS_GROUPS=assets assets:precompile